code/docker/buildLibraries

```diff
@@ -0,0 +1,19 @@
+#!/bin/ash
+set -e
+
+BASE=/opt/build
+LIBRARIES="common,service-chassis"
+
+for library in $(echo $LIBRARIES | sed "s/,/ /g") ;
```

Comment:
Good idea

Reply:
Wonder if there is some babel magic we could use. Something for another time.

Reply:
Babel magic to do which bit?